bikesharing/migrations/0020_locationtracker_internal_only.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.4 on 2020-03-19 10:09
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bikesharing', '0019_auto_20200317_2042'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='locationtracker',
+            name='internal_only',
```

Comment:
personal naming opinion: I would drop the `_only` there.

Reply:
fixed in 9d62f3f